.circleci/config.yml

```diff
@@ -22,11 +22,6 @@ jobs:
           command: |
             ./scripts/go-mod-download.sh
       - run: "! go fmt ./... 2>&1 | read"
-      - run:
-          name: golint
-          command: |
-            go get golang.org/x/lint/golint
-            golint ./...
 #      - run:
 #          name: gosec
 #          command: |
```

Comment:
gosec can go away too, please remove it.

Reply:
thanks, fixed